client/src/pages/SignUp.js

```diff
@@ -0,0 +1,30 @@
+import "./SignUp.css";
+import Card from "../components/Card";
+import BackButton from "../components/BackButton";
+import FooterSubmit from "../components/Card_components/FooterSubmit";
+import InputField from "../components/InputField";
+import { Link } from "react-router-dom";
+
+export default function SignUp() {
+  return (
+    <main id="Signup" className="card-screen">
+      <Card header={<BackButton />} footer={<FooterSubmit text="Sign Up" />}>
+        <h2 className="margin-b--s">Sign Up</h2>
+        <p>
+          Already have an account?
+          <br />
+          <Link className="opaque link" to="/login">
+            Login instead
+          </Link>
+        </p>
+
+        <form className="dispFlex margin-b--l">
+          <InputField id="Signup" label="Username" />
+          <InputField id="Signup" label="Email" />
+          <InputField id="Signup" label="Password" type="password" />
+          <InputField id="Signup" label="Repeat password" type="password" />
```

Comment:
Hey Jakob, I think IDs should be unique and only used once. also in connection with the input a bit confusing. you use it for styling here, right?

Reply:
Hey I can see where it get's confusing, it's actually not an id for an Element it's a prop that I am using within my Input to make the labels work in accordance with the input fields. 

I'll rename the property to address that

Reply:
ah, got that! sorry. makes sense now!

Reply:
No it's definitely a valid concern :) Thank you for your input